#------------- Rocket parameters --------------------
#
# Defines parameters specific to the rocket:
#   - General dimensions of the components
#   - Rocket engine specifications
#   - Targeted apogee
#
# ----------------------------------------------------
stages: 4
diameters:
  - 0
  - 0.156 
  - 0.156 
  - 0.135
stage_z:
  - 0 
  - 0.242 
  - 3.43 
  - 3.51 
fin_n: 4
fin_top_offset: 3.15 
fin_span: 0.17
fin_root_chord: 0.28 # base length
fin_tip_chord: 0.125
fin_sweep: 0.107
fin_thickness: 0.004
lug_n: 2
lug_S: 5.7e-4 
dry_mass: 34.186 # Dry rocket mass + ABSÂ leftover
dry_CM: 1.77
dry_I:
  - 39
  - 39
  - 2
propellant_mass: 4.228
propellant_CM: 2.5
Isp: 207
maxThrust:
  - 100
  - 100
  - 3000
minThrust: 
  - -100
  - -100
  - 0
minTorque: -50
maxTorque: 50
Cd:
  - 2
  - 2
  - 0.335
initial_speed: 0